apps/wallet/src/background/keyring/Vault.test.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { describe, expect, it, vi } from 'vitest';
+
+import { Vault } from './Vault';
+import { testVault } from '_src/test-utils/vault';
+
+describe('Vault', () => {
+    it('initializes', () => {
+        const vault = new Vault(testVault.entropy, [...testVault.keypairs]);
+        expect(vault).toBeDefined();
```

Comment:
I don't think it's technically possible for this to not be the case, I think if anything you'd just want this to be an expect not to throw assertion, but that should be tested through the other cases so I think you can just remove this test entirely.